test/v2/models/response/createAmendHistoricNonFhlUkPropertyAnnualSubmission/CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionResponseSpec.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.createAmendHistoricNonFhlUkPropertyAnnualSubmission
+
+import play.api.libs.json.{ JsObject, JsValue, Json }
+import support.UnitSpec
+
+class CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionResponseSpec extends UnitSpec {
+
+  private val model = CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionResponse(
+    transactionReference = Some("0000000000000001")
+  )
+
+  val desJson: JsValue = Json.parse("""{
```

Comment:
can you rename `desJson` to `downstreamJson` - it's an IFS endpoint but I think the general 'downstream' would be best